src/components/Scroller.tsx

```diff
@@ -0,0 +1,24 @@
+import { HTMLProps } from 'react'
+
+interface ScrollerProps {
+  scrollbars?: 'vertical' | 'horizontal' | 'both'
+}
+
+export function Scroller({
+  scrollbars = 'both',
+  ...props
+}: ScrollerProps & HTMLProps<HTMLDivElement>) {
+  return (
+    <div
+      {...props}
+      style={{
+        ...props.style,
+        ...(scrollbars === 'vertical' && { overflowY: 'auto' }),
+        ...(scrollbars === 'horizontal' && { overflowX: 'auto' }),
+        ...(scrollbars === 'both' && { overflow: 'auto' }),
+        scrollbarWidth: 'thin',
```

Comment:
This doesn't work on Mac, at least with the default settings when scrollbars are only displayed when interacted with

Reply:
https://github.com/user-attachments/assets/04ad3efc-b079-40fd-a06b-58e5f79e320e